tests/test_examples.py

```diff
@@ -16,26 +16,34 @@
 
 from gordo_components.data_provider.providers import DataLakeProvider
 from gordo_components.dataset.datasets import TimeSeriesDataset
+from gordo_components.dataset.sensor_tag import SensorTag
 
 logger = logging.getLogger(__name__)
 
+CONFIG = dict(
+    from_ts=dateutil.parser.isoparse("2014-07-01T00:10:00+00:00"),
+    to_ts=dateutil.parser.isoparse("2015-01-01T00:00:00+00:00"),
+    tag_list=[SensorTag(name=f"Tag {i}", asset=None) for i in range(10)],
+)
+
 
 def _fake_data():
-    data = pd.DataFrame({f"sensor_{i}": np.random.random(size=100) for i in range(10)})
+    index = pd.date_range(CONFIG["from_ts"], CONFIG["to_ts"], freq="12T")
```

Comment:
Is this because it fails when they the index is _not_ a datetime? Do you think it's worth it to make it such that it'll still work without datetime indexes?

Reply:
Yes, correct-a-mundo. It appeared to me as though they were intended to have datetime indices. Is that not the case?

Reply:
Now it runs with a regular 'ole index.